integreat_cms/cms/views/pages/page_sbs_view.py

```diff
@@ -71,6 +71,14 @@ def get(self, request, *args, **kwargs):
         source_page_translation = page.get_translation(source_language.slug)
         target_page_translation = page.get_translation(target_language.slug)
 
+        # For the text diff, use the latest source translation that was created before the target translation
+        filter_args = {"language": source_language}
+        if target_page_translation:
+            filter_args["last_updated__lte"] = target_page_translation.last_updated
+        source_previous_translation = (
+            page.translations.filter(**filter_args).order_by("-last_updated").first()
+        )
```

Comment:
```suggestion
        old_translation_content = ""
        if target_page_translation:
            source_previous_translation = (
                page.translations.filter(
                    language=source_language,
                    last_updated__lte=target_page_translation.last_updated,
                )
                .order_by("-last_updated")
                .first()
            )
            if source_previous_translation:
                old_translation_content = source_previous_translation.content
```
If no target_page_translation exists, I would argue that the old content should be empty and the diff is "everything"...